pkg/operator/edgefs/cluster/utils.go

```diff
@@ -281,7 +281,9 @@ func (c *cluster) resolveNode(nodeName string) *rookv1alpha2.Node {
 }
 
 func (c *cluster) LabelTargetNode(nodeName string) {
-	c.AddLabelsToNode(nodeName, map[string]string{c.Namespace: "cluster"})
+	if err := c.AddLabelsToNode(nodeName, map[string]string{c.Namespace: "cluster"}); err != nil {
+		logger.Errorf("failed to add labels to node %q. %v", nodeName, err)
```

Comment:
@sabbot Is logging this error sufficient, or should these errors be returned to the caller? Also please check the other error handling for edgefs in this PR, thanks.

Reply:
@sabbot Please review.